### DISCUSSION TITLE
Ignore uppercase mode

### DISCUSSION THREAD

**Discussion by F4400** [monkeytypegame/monkeytype#3559] [Ideas]
German here. All our nouns are capitalized. Sometimes I don't want to press the shift key but still write in my mother tongue. Would be nice if there was a mode/toggle/setting so that you can ignore the case.

**Comment by F4400** [comment]
Or additionally a filter for uppercase words.

**Comment by Miodec** [comment]
The second i remove all upper case words from tests my inbox gets destroyed by germans saying their nouns are capitalized. When I revert the change I get complaints about the capital letters. I guess there is no way to please everybody.

**Reply by F4400** [reply]
That's why I suggested to add a mode or switch that gives you the possibility to change this. I never said that for everyone all nouns should be lowercase but that you have the choice to decide about it.
